### ISSUE TITLE
Audio Muting for Nlziet

### ISSUE THREAD

**Issue by TheodorusL** [FrostCo/AdvancedProfanityFilter#523]
#### :movie_camera: The Site
app.nlziet.nl

#### :speech_balloon: Subtitle Element/Node
```html
<section class="VTn6M">
  <div class="FxxsA">
    <div>
      <span>
        <span class="white" style="color:white;">
          En hij is er inmiddels achter
        </span>
        <br>
        <span class="white" style="color:white;">
          waar je woont.
        </span>
      </span>
    </div>
  </div>
</section>
```

#### :pencil: Additional notes
The active subtitle on this frame contains two lines:

_En hij is er inmiddels achter
waar je woont._

I hope you can help me to add NLZiet as an Audio Muting Site. 
I've tried to add it myself, but could not get it working. 
Thank you very much for helping me!

**Comment by richardfrost** [comment]
Hi @TheodorusL, thanks for reaching out about this. Unfortunately I haven't been able to access a video on the site without a subscription, and because I'm in the United States it doesn't seem like I can setup an account.

If you're willing to help me gather some information it might be possible to add support for this site. If you'd like to help out, please send an email to support @ frostco.dev (without the spaces).

If not, that's okay. I'll see if I can figure anything else out.

**Comment by TheodorusL** [comment]
Hi Richard,

Thanks for wanting to help me out.
I can give you my personal login:

Username: ***@***.***
Password: LH3WrZk#gvcF&n2A5x2PtK3o

I used this series for testing: https://app.nlziet.nl/vod/jBEBQUkg_0i3rfaasyIDVw
The first subtitles in this episode are shown after a few seconds. I used the word 'woont' as trigger to test.

Thanks again for helping me out!

With kind regards,

Theodorus


------ Oorspronkelijke bericht ------
Van "Richard Frost" ***@***.******@***.***>>
Aan "FrostCo/AdvancedProfanityFilter" ***@***.******@***.***>>
CC "Theodorus" ***@***.******@***.***>>; "Mention" ***@***.******@***.***>>
Datum 10-11-2023 01:11:41
Onderwerp Re: [FrostCo/AdvancedProfanityFilter] Custom Audio Site (Issue #523)


Hi @TheodorusL<https://github.com/TheodorusL>, thanks for reaching out about this. Unfortunately I haven't been able to access a video on the site without a subscription, and because I'm in the United States it doesn't seem like I can setup an account.

If you're willing to help me gather some information it might be possible to add support for this site. If you'd like to help out, please send an email to support @ frostco.dev (without the spaces).

If not, that's okay. I'll see if I can figure anything else out.

**Comment by richardfrost** [comment]
Hi @TheodorusL, sorry for the delay. I've have other things I needed to focus on for a bit. Thanks for being willing to help out. Credentials can't be shared on Github, so could you please send them to the email address mentioned in my last post?

**Comment by richardfrost** [comment]
@TheodorusL I'm so sorry this has taken me so long. I am usually quite a bit faster than this, but have just been really busy these last few weeks.

Would you be able to test this config?

```json
{
  "app.nlziet.nl": [
    {
      "className": "VTn6M",
      "displaySelector": "section.VTn6M",
      "mode": "element",
      "subtitleSelector": "span",
      "tagName": "SECTION",
      "videoSelector": "div.bitmovinplayer-container video"
    },{
      "displaySelector": "section.VTn6M",
      "mode": "elementChild",
      "parentSelector": "section.VTn6M",
      "tagName": "SPAN",
      "videoSelector": "div.bitmovinplayer-container video"
    }
  ]
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

**Comment by TheodorusL** [comment]
Thank you very much. I did some testing, and it works great! Thank you very, very much!

With kind regards,

Theodorus


------ Oorspronkelijke bericht ------
Van "Richard Frost" ***@***.******@***.***>>
Aan "FrostCo/AdvancedProfanityFilter" ***@***.******@***.***>>
CC "Theodorus" ***@***.******@***.***>>; "Mention" ***@***.******@***.***>>
Datum 8-12-2023 01:51:32
Onderwerp Re: [FrostCo/AdvancedProfanityFilter] Audio Muting for Nlziet (Issue #523)


@TheodorusL<https://github.com/TheodorusL> I'm so sorry this has taken me so long. I am usually quite a bit faster than this, but have just been really busy these last few weeks.

Would you be able to test this config?

{
  "app.nlziet.nl": [
    {

"className": "VTn6M",
      "displaySelector": "section.VTn6M",
      "mode": "element",
      "subtitleSelector": "span",
      "tagName": "SECTION",
      "videoSelector": "div.bitmovinplayer-container video"
    },{
      "displaySelector": "section.VTn6M",
      "mode": "elementChild",
      "parentSelector": "section.VTn6M",
      "tagName": "SPAN",
      "videoSelector": "div.bitmovinplayer-container video"
    }
  ]
}


To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

**Comment by richardfrost** [comment]
You're very welcome! Thanks for being willing to help make it possible. Take care!

**Comment by richardfrost** [comment]
You're very welcome @TheodorusL! Thanks for being willing to help make it possible. Enjoy!

**Comment by richardfrost** [comment]
This has been included in version [3.3.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/Version). Feel free to comment here if you have any feedback for this update.
